spring-security-karaf-login-module/src/main/java/org/pentaho/platform/osgi/auth/spring/SpringSecurityLoginModule.java

```diff
@@ -181,13 +182,30 @@ public boolean login() throws LoginException {
     // If they have AdministerSecurity, grant the Karaf admin role
     if ( getAuthorizationPolicy().isAllowed( AdministerSecurityAction.NAME ) ) {
       principals.add( new RolePrincipal( KARAF_ADMIN ) );
```

Comment:
This is no longer needed and can be removed together with the `SpringSecurityLoginModule.KARAF_ADMIN` constant.